sdk/openai/azure-ai-openai/src/main/resources/META-INF/native-image/com.azure/azure-ai-openai/reflect-config.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "name": "com.azure.storage.blob.implementation.OpenAIClientImpl",
```

Comment:
This package name doesn't look right. It's using `storage.blob`. Also, do we need to include all the models that are reflectively accessed in this configfile?

Reply:
Argh, and as I tested this proves this is probably useless from the beginning... I'll totally remove it. My main issues where with the proxies.